### PR TITLE
Add label translation & some clean up

### PR DESCRIPTION
**Pull request by mahbx**
# Changes :gem:
* API now translates labels into Norwegian before sending to FE
* Change a few exception returns into exception raises
* Add tests for translation function and labels table in db
* Edit some documentation

### CODE REVIEW

File: src/test/test_db.py [hunk 1]
```diff
@@ -196,4 +195,45 @@ def test_clear_table():
...
+        for i in range(5):
+            result = models.get_n_labels(i)
+            assert len(result) == i
+
+    assert result
+
+
+def test_get_n_labels_bad_reqeust():
+    """
+        Test that get_n_labels raises exeption if n is larger than number of labels
```
Comment (andreacx) [inline src/test/test_db.py:214]: too long, hvis det er lov å si;))

File: src/webapp/models.py [hunk 2]
```diff
@@ -351,7 +351,8 @@ def drop_table(table):
 
 def seed_labels(app, filepath):
     """
-        Read file in filepath and upload to database
+        Read file in filepath and upload to database. This function is used to update labels in database.
```
Comment (andreacx) [inline src/webapp/models.py:354]: toooooo long

File: src/test/test_db.py (hunk 1 above)
Comment (mahbx) [reply src/test/test_db.py:214]: lov å si men hva mener du??

File: src/webapp/api.py [hunk 3]
```diff
@@ -80,16 +80,17 @@ def get_label():
...
+        "label": norwegian_label
     }
     return json.dumps(data), 200
 
 
 @app.route("/classify", methods=["POST"])
 def classify():
     """
-        Classify endpoint for continious guesses.
+        Classify endpoit for continuous guesses.
```
Comment (joaaax) [inline src/webapp/api.py:93]: Typo

**Pull request merged**